### DISCUSSION TITLE
Hasil Accuracy Knit-HTML dan RMD berbeda

### DISCUSSION THREAD

**Discussion by rachmat-kimung** [teamalgoritma/community#367] [Q&A] [answered]
Saya mendapatkan perbedaan hasil accuracy antara yang ditampilkan oleh RMD dan setelah hasil saya knit kedalam HTML

Accuracy pada rmd :

1. 
![image](https://user-images.githubusercontent.com/79957356/193231054-badf7f56-c510-42d0-b774-49120450c816.png)

Accuracy pada knit html : 
2.
![image](https://user-images.githubusercontent.com/79957356/193231332-6c041cfa-2bc3-4964-bb1d-e6d7acd3ddb8.png)

mungkin ada yang dapat memberikan solusi terkait masalah ini

**Comment by kevbow** [comment, marked as answer]
Halo @rachmat-kimung,

Sebelumnya, apakah Anda telah menggunakan set.seed() ketika melakukan cross validation?

**Reply by rachmat-kimung** [reply]
set.seed(100) sudah saya gunakan

**Reply by VictorNugraha** [reply]
Halo @rachmat-kimung ,

Mungkin bisa dicoba untuk melakukan beberapa step berikut ini:

- Step 1: Membersihkan environment pada R-Studio
- Step 2: Menambahkan sebuah parameter pada chunk yang menampung code perhitungan matrix accuracy. Berikut contohnya:
![image](https://user-images.githubusercontent.com/28379954/193242320-08953093-8626-47ce-8a2a-b13cd54a6411.png)

Parameter tersebut diperuntuhkan untuk menghentikan evaluasi ulang.

Semoga dengan 2 tahapan berikut, dapat membantu menyelesaikan permasalahan yang ada.

Terima kasih.

**Reply by rachmat-kimung** [reply]
terima kasih sarannya, sayangnya masih berbeda hasilnya antara RMD dan Knit-HTML

**Reply by jafarhf1** [reply]
Halo @rachmat-kimung, mungkin bapak bisa memastikan penggunaan set.seed() di lokasi yang tepat seperti misalnya sebelum dilakukan cross validation dan juga sebelum dilakukan proses fitiing.

Semoga dapat membantu.

Terima kasih.
